resources-limit/env0.rego

```diff
@@ -1,26 +1,25 @@
 package env0
 
 # METADATA
-# title: only admin can approve more than 5 planned resources
+# title: requires approval for more than 5 planned resources
 # description: require approval on more than 5 planned resources
 pending[format(rego.metadata.rule())] {
-	input.plan.planned_values.resources > 5
-	input.requestingUser.role != "ADMIN"
-	count(input.approvers) > 1
+	count(input.plan.resource_changes) >= 5
+	count(input.approvers) == 0
```

Comment:
@tomer-landesman last question
so say 7 resources are changed. at first we have 0 approvers, so both conditions would be true and result will be pending.
After an approval this rule will be ignored? as `count(approvers) != 0` ? or what will be the result after 1 approvals?

Reply:
after an approval, the first rule won't evaluate, because it an AND gate between the conditions. the `resources < 5` also won't be evaluated, so the only thing left is the `approvers >= 1` which will result in `allow`